clients/admin-ui/src/features/dataset/EditDatasetDrawer.tsx

```diff
@@ -21,8 +21,17 @@ const EditDatasetDrawer = ({ dataset, isOpen, onClose }: Props) => {
     if (dataset) {
       const updatedDataset = { ...dataset, ...values };
       try {
-        await updateDataset(updatedDataset);
-        toast(successToastParams("Successfully modified dataset"));
+        const result = await updateDataset(updatedDataset);
+        // TODO: we should systematically coerce errors into their types (see fidesops)
```

Comment:
I went looking for a related info fidesops but couldn't pin down what you meant. Maybe create an issue to document or implement this.